SECURITY.md

```diff
@@ -7,9 +7,18 @@
  - :white_check_mark: (please double check this is consistent with the [CVE process](https://pivotal.io/security), we share this one with Spring)
  
  ## Supported Versions
+ ### Reactor Core
```

Comment:
This could be merged together by mentionning the BOM name:

| Version | Supported          |
| ------- | ------------------ |
| Dysprosium Train (core 3.3.x, netty 0.9.x)   | :white_check_mark: |
| Califonium Train (core 3.2.x, netty 0.8.x)   | :white_check_mark: |
| Bismuth and below (core < 3.2, netty < 0.8)   | :x: |
| Reactor 1.x and 2.x Generations  | :x:                |

Reply:
done